solr/core/src/test/org/apache/solr/handler/component/DebugComponentTest.java

```diff
@@ -52,7 +64,7 @@ public void testBasicInterface() {
         "//str[@name='querystring']='*:*'",
         "//str[@name='parsedquery']='MatchAllDocsQuery(*:*)'",
         "//str[@name='parsedquery_toString']='*:*'",
-        "count(//lst[@name='explain']/*)=3",
+        "count(//lst[@name='explain']/*)=6",
```

Comment:
How does this PR affect this assertion of an existing test?

Reply:
I added more docs in the setup of the test class, for the new debug tests I added.
I thought that changing the few lines of debug that were counting the total docs was not much of a big deal.
I can obviously change that if you think it is worth.

Reply:
I suppose I'd need to see a before & after to truly "see" the change.